### PR TITLE
Mention Data Centre (with placeholder text)

### PR DESCRIPTION
**Pull request by mashehu**
Not sure what text to write, but needed the placeholder to check alignment of logo.

<a href="https://gitpod.io/#https://github.com/nf-core/nf-co.re/pull/1556"></a>

### CODE REVIEW

**Comment by mashehu** [comment]
@nf-core-bot fix linting

File: markdown/about.md [hunk 1]
```diff
@@ -57,6 +57,14 @@ If you work with a project that would benefit from an explicit link with nf-core
 
 Finally, we would specifically like to acknowledge and thank the following sponsors who help to support the project:
 
+### SciLifeLab Data Centre
+
+<img src="/assets/img/contributors-colour/SciLifeLabDC.svg" alt="SciLifeLab Data Centre" class="float-end darkmode-image me-5 mb-5 w-25 ms-3">
+
+Iste officiis dolor necessitatibus pariatur aut. Voluptas unde optio deleniti in dolorem iste aliquam error. Et est ipsa quisquam voluptas.
+
+Saepe qui molestiae molestiae earum. Tempore ut voluptas velit. Architecto doloremque quasi autem rerum expedita iste et. Consequatur voluptatem beatae deleniti esse dolorum minima. Esse quas quos distinctio.
```
Comment (maxulysse) [inline markdown/about.md:66]: ?

Comment (maxulysse) [reply markdown/about.md:66]: I'm sure you can find a better description of the Data Centre

**Review by maxulysse — APPROVED** [review]

File: markdown/about.md [hunk 2]
```diff
@@ -85,6 +77,17 @@ Amongst other things, the CZI EOSS grant money has enabled the nf-core community
...
+The CZI grant employs personnel based at the [SciLifeLab National Genomics Infrastructure](https://ngisweden.scilifelab.se/), the [Quantitative Biology Center](http://qbic.life/) in Tübingen Germany, and [Seqera Labs](https://seqera.io/).
+
+<div class="clearfix"></div>
+
+### SciLifeLab Data Centre
+
+<img src="/assets/img/contributors-colour/SciLifeLabDC.svg" alt="SciLifeLab Data Centre" class="float-end darkmode-image me-5 mb-5 w-25 ms-3">
+
+The [SciLifeLab Data Centre](https://www.scilifelab.se/data/) supports nf-core with funding from the [SciLifeLab & Wallenberg National Program for Data-Driven Life Science](https://www.scilifelab.se/data-driven/).
+This funding covers a full-time position ([@mashehu](https://github.com/mashehu), Matthias Hörtenhuber) to work on maintainance of nf-core framework code. For example, the nf-core website and the nf-core/tools package.
```
Comment (mashehu) [inline markdown/about.md:89]:
```suggestion
This funding covers a full-time position ([@mashehu](https://github.com/mashehu), Matthias Hörtenhuber) to work on maintainance of nf-core framework code. For example, the [nf-core website](https://nf-co.re/) and the [nf-core/tools package](https://github.com/nf-core/tools/).
```

**Pull request merged**